app/helpers/dashboard/protocols_helper.rb

```diff
@@ -64,4 +64,14 @@ def display_requests_button(protocol, access)
       end
     end
   end
+
+  def display_merges(protocol)
```

Comment:
I think we should probably add a `has_many :protocol_merges` association to the Protocol model then you can replace this helper method with `protocol.protocol_merges.pluck(:merged_protocol_id)` which will return the merged protocol IDs. Without the association it would be `ProtocolMerge.where(master_protocol: protocol).pluck(:merged_protocol_id)`